packages/terra-overlay/src/OverlayContainer.jsx

```diff
@@ -7,24 +7,30 @@ import styles from './Overlay.module.scss';
 const cx = classNames.bind(styles);
 
 const propTypes = {
-/**
- * The overlay and the elements the overlay should hide when overlay isRelativeToContainer and isOpen.
- */
+  /**
+  * The Overlay and the content to be displayed within the overlay.
+  */
+  overlay: PropTypes.node,
```

Comment:
Should this only accept overlay or loading overlay elements?

Reply:
this prop can be used for both overlay or loading overlay.  the content which is to be displayed on overlay or loading overlay will be passed as overlay to Overlay Container.

Reply:
oh I was thinking
```
overlay: PropTypes.oneOf([PropTypes.instanceOf(Overlay), PropTypes.instanceOf(LoadingOverlay)]),
```

Reply:
this requires importing of Overlay and LoadingOverlay which will violate the Lint rule : [import/no-cycle](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-cycle.md). still we can bypass this rule error by doing it in this way 
`overlay: PropTypes.oneOf([PropTypes.instanceOf('./Overlay'), PropTypes.instanceOf('./LoadingOverlay')]),`  but am not sure is it preferable to do it in this way or not.

Reply:
yeah lets not override the linter here. The cyclical dependency issues can get weird